### PR TITLE
Add a troubleshooting doc

### PR DESCRIPTION
**Pull request by Fedosin**

### CODE REVIEW

File: docs/user/troubleshooting.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+## Common troubleshooting techniques
+
+All Cluster Cloud Controller Manager Operator (CCCMO) components are deployed in `openshift-cloud-controller-manager` namespace. You can check their status by executing `oc get all -n openshift-cloud-controller-manager`.
+
+To see the operator logs use `oc logs -n openshift-cloud-controller-manager cluster-cloud-controller-manager-operator-<random suffix> -c cluster-cloud-controller-manager`.
+
+## Cluster Cloud Controller Manager Operator is degraded
+
+First, check the cluster operator status with `oc get clusteroperator cloud-controller-manager -oyaml` to find potential problems. Most likely the operator will have `Degraded` condition set to True.
```
Comment (JoelSpeed) [inline docs/user/troubleshooting.md:11]: Nit, add a gap between the flag and argument so it's clear `yaml` is an argument
```suggestion
First, check the cluster operator status with `oc get clusteroperator cloud-controller-manager -o yaml` to find potential problems. Most likely the operator will have `Degraded` condition set to True.
```

File: docs/user/troubleshooting.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+In parallel with that CCCMO also monitors the `kubecontrollermanager` resource. It doesn't provision any Cloud Controller Manager related resources until the condition is set to False there.
+
+When all KCM pods are restarted and KCM operator informs that it doesn't own cloud controllers anymore by setting the condition, CCCMO starts working. It deploys all necessary resources, monitors their statuses, and, if everything is fine, sets the same condition to True in its cluster operator resource. To checks this, execute `oc get clusteroperator cloud-controller-manager -oyaml` to see the value of `CloudControllerOwner` condition.
+
+When CCCMO sets the condition, the migration is done. Normally it make take up to 15 minutes.
```
Comment (JoelSpeed) [inline docs/user/troubleshooting.md:25]: Where did the 15 minute value come from?
```suggestion
When CCCMO sets the condition, the migration is done. We expect this to take around 15 minutes.
```

File: docs/user/troubleshooting.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+# Troubleshooting guide
+
+## Common troubleshooting techniques
+
+All Cluster Cloud Controller Manager Operator (CCCMO) components are deployed in `openshift-cloud-controller-manager` namespace. You can check their status by executing `oc get all -n openshift-cloud-controller-manager`.
+
+To see the operator logs use `oc logs -n openshift-cloud-controller-manager cluster-cloud-controller-manager-operator-<random suffix> -c cluster-cloud-controller-manager`.
```
[the remark this answers is not in the document]
Comment (elmiko) [reply docs/user/troubleshooting.md:7]: i think it would be good to call out this distinction in the opening paragraph

**Review by elmiko — COMMENTED** [review]
i think the content looks nice, i generally agree with the other comments.

**Pull request merged**